website/src/docs/getting_started.mdx

```diff
@@ -17,32 +17,37 @@ The easiest way to install <Garn/> is with the following command:
 
 You can look at the contents of the script [here](../install.sh).
 
-<Garn/> requires [Nix](https://nixos.org/). If you don't have Nix, the above command will install it for you.
+<Garn /> requires [Nix](https://nixos.org/). If you don't have Nix, the above
+command will install it for you.
 
 ### Manual installation
 
 If you prefer, you can also manually do the steps that the above script automates. Those are:
 
 1. **Install Nix**. To install Nix, follow the instructions [here](https://nixos.org/download).
 2. **Configure Nix to use the <Garnix/> cache**. You can do that by adding the lines:
-  ```
+
+```
 extra-substituters = https://cache.garnix.io
 extra-trusted-public-keys = cache.garnix.io:CTFPyKSLcx5RMJKfLo5EEPUObbA78b0YQ2DTCJXqr9g=
-  ```
-  To `/etc/nix/nix.conf` (note that you'll need sudo).
+```
+
+To `/etc/nix/nix.conf` (note that you'll need sudo).
```

Comment:
Maybe doesn't need to be now, but just occurred to me that we need to tell users to restart the daemon after this change.

Reply:
Good point! I think we don't though. At least `nix show-config` shows the the cache without restarting. Although we should make sure that the cache is actually being used.

Reply:
Yeah, never mind. `nix show-config` is just lying.